src/fillTank.test.js

```diff
@@ -1,11 +1,109 @@
 'use strict';
 
 describe('fillTank', () => {
-  // const { fillTank } = require('./fillTank');
+  const { fillTank } = require('./fillTank');
 
-  it('should ', () => {
+  it('should be declared', () => {
+    expect(fillTank)
+      .toBeInstanceOf(Function);
+  });
+
+  it(`should order full tank if 'amount' is not given`, () => {
+    const customer = {
+      money: 3000,
+      vehicle: {
+        maxTankCapacity: 40,
+        fuelRemains: 8,
+      },
+    };
+    const expectedResult = 40;
+
+    fillTank(customer, 4);
```

Comment:
The test case description suggests that the 'amount' is not given, but the function call `fillTank(customer, 4)` includes an 'amount' parameter. Ensure that the test case aligns with the description by either removing the 'amount' parameter or adjusting the description.